isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -24,30 +24,124 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
 -->
   <h2>Overview</h2>
 
+  <p>
+    This installation guide is for ISIS3 users interested in installing ISIS3 through conda, or compiling on their own systems using cmake.  
+    If you are a developer, we refer you to our <a href="https://github.com/USGS-Astrogeology/ISIS3">GitHub repository.</a>
+  </p>
+
+
+  <h3>ISIS3 Installation With Conda</h3>
+    <ol>
+      <li>Download either the Anaconda or Miniconda installation script for your OS platform.  Anaconda is a much larger distribtion of packages supporting scientific python, while Miniconda is a minimal installation and not as large: 
+      <a href="https://www.anaconda.com/download">Anaconda installer</a>, <a href="https://conda.io/miniconda.html">Miniconda installer</a></li>
+      <li>Open a terminal window in the directory where you downloaded the script, and run the following commands (in this example, we chose to do a full install of Anaconda):</li>
+        <pre>
+          chmod +x Anaconda3-5.2.0-Linux-x86_64.sh
+          ./Anaconda3-5.2.0-Linux-x86_64.sh          
```

Comment:
This command is OS-specific. Might want to mention that this is an example for the Linux platform.

Reply:
Good catch!